Source/Core/Core/PowerPC/PPCTables.h

```diff
@@ -7,50 +7,51 @@
 #include "Core/PowerPC/Gekko.h"
 #include "Core/PowerPC/Interpreter/Interpreter.h"
 
+// Flags that indicate what an instruction can do.
 enum
 {
-	FL_SET_CR0         = (1<<0), //
-	FL_SET_CR1         = (1<<1), //
-	FL_SET_CRn         = (1<<2), //
-	FL_SET_CRx         = FL_SET_CR0 | FL_SET_CR1 | FL_SET_CRn, //
-	FL_SET_CA          = (1<<3), // carry
-	FL_READ_CA         = (1<<4), // carry
-	FL_RC_BIT          = (1<<5),
-	FL_RC_BIT_F        = (1<<6),
-	FL_ENDBLOCK        = (1<<7),
-	FL_IN_A            = (1<<8),
-	FL_IN_A0           = (1<<9),
-	FL_IN_B            = (1<<10),
-	FL_IN_C            = (1<<11),
-	FL_IN_S            = (1<<12),
+	FL_SET_CR0         = (1<<0),  // Sets CR0.
+	FL_SET_CR1         = (1<<1),  // Sets CR1.
+	FL_SET_CRn         = (1<<2),  // Encoding decides which CR can be set.
+	FL_SET_CRx         = FL_SET_CR0 | FL_SET_CR1 | FL_SET_CRn,
+	FL_SET_CA          = (1<<3),  // Sets the carry flag.
+	FL_READ_CA         = (1<<4),  // Reads the carry flag.
+	FL_RC_BIT          = (1<<5),  // Sets the record bit.
+	FL_RC_BIT_F        = (1<<6),  // Sets the record bit. Used for floatig point instructions that do this.
+	FL_ENDBLOCK        = (1<<7),  // Specifies the end of a JIT block.
```

Comment:
Correction: this instruction *can* end the JIT block, that is, it can be an exit point. It doesn't have to.